<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sinusoidal Model Spectrogram Analyzer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="input_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Input File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compute_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>89</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
